locales/pt/validation-inline.php

```diff
@@ -20,7 +20,7 @@
     'alpha_dash'           => 'Este campo pode conter apenas letras, números, traços e o caractere de sublinhado.',
     'alpha_num'            => 'Este campo pode conter apenas letras e números',
     'array'                => 'Este campo deve ser uma matriz.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'Este campo já está ligado.',
```

Comment:
I believe it should 'Este campo já está anexado.' just like in locales/pt/validation.php

Reply:
the rest is ok for you ?